src/garage/torch/algos/vpg.py

```diff
@@ -307,20 +407,23 @@ def process_samples(self, itr, paths):
 
         Returns:
             tuple:
```

Comment:
returning a tuple is equivalent to multiple return values. that is

```python
def foo():
    return tuple(1, 2)

def bar():
    return 1, 2

def baz():
    return (1, 2)

assert foo() == bar() == baz()
```
successfully executes.

that means that we should document this method as a multiple-return method. that has its own style which you can reference here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

in brief, it looks like the `Args:` section but without names:

```python
def foo(bar, baz):
    """Apply foo to bar and baz.

    Args:
        bar (int): A bar.
        baz (int): A baz.

    Returns:
        int: 1
        int: 2
    """
    return 1, 2